site/setup.cfg

```diff
@@ -3,6 +3,14 @@
 name = cds-rdm
 version = attr: cds_rdm.__version__
 
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=3.6
```

Comment:
I think 3.7 is reaching EOL and security fixes, maybe upgrade to 3.8? or even 3.9 since the RDM images with AlmaLinux are 3.9 only?